extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -37,21 +37,23 @@ const metadataProvider = classes.MetadataProvider;
 
 /**
  *
- * @param {string} name - Data source name
- * @param {string} wadoUriRoot - Legacy? (potentially unused/replaced)
- * @param {string} qidoRoot - Base URL to use for QIDO requests
- * @param {string} wadoRoot - Base URL to use for WADO requests
- * @param {boolean} qidoSupportsIncludeField - Whether QIDO supports the "Include" option to request additional fields in response
- * @param {string} imageRengering - wadors | ? (unsure of where/how this is used)
- * @param {string} thumbnailRendering - wadors | ? (unsure of where/how this is used)
+ * @param {string} baseUrl - The root URL for DICOMWeb calls
```

Comment:
Have the names matche the dicomweb client definition - that is, it should take url as the base url parameter, and ONLY add wadoURLPrefix etc if specified.